apps/backend/core/lifecycle.py

```
@@ -12,16 +12,105 @@
+
+    logger.info("Running configuration validation...")
+
+    # Get configuration (use existing settings for now)
+    config = settings
+
+    # Run validation (show_table=False for production environments)
+    report = await validate_startup(config, show_table=False)
+
+    # Store report in app state for health endpoints
+    app.state.validation_report = report
+    app.state.validation_timestamp = datetime.utcnow()
```

Ruff (0.14.4)

`datetime.datetime.utcnow()` used

(DTZ003)